CMakeLists.txt

```diff
@@ -417,7 +417,7 @@ find_package(LibXml2 REQUIRED)
 
 add_library(openrave_gpgme IMPORTED INTERFACE)
 if (OPT_ENCRYPTION)
-  find_package(Gpgmepp 1.14 REQUIRED)
+  find_package(Gpgmepp 1.13 REQUIRED)

```

Comment:
github actions run on Ubuntu, not Debian. To conform Ubuntu 20, I needed to downgrade Gpgmepp required version. (actually why it is 1.14? least version should be specified..)